addon/templates/components/materialize-tabs-tab.hbs

```diff
@@ -0,0 +1,3 @@
+{{#link-to dest}}
```

Comment:
**Note to self**: Need to expose ability to add classes, etc... to the `<a>` tag